cfgov/v1/jinja2/v1/_includes/molecules/half-width-link-blob.html

```diff
@@ -0,0 +1,39 @@
+{# ==========================================================================
+
+   half-width-link-blob.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a half-width-link-blob molecule when given:
+
+   settings:
+
+      heading: The text to include in the heading
+
+      content: The content to display below the heading (typically a paragraph)
+
+      links: A tuple of items to create a list of links. Objects include a url
```

Comment:
Why do this vs breaking out two individual `link_text` and `link_url` entries? Is this how it would come from the CMS? Having two levels of nested objects requires more examination of the docs and code to implement this, which could more easily introduce errors.